src/nvim/options.lua

```diff
@@ -8629,6 +8643,7 @@ local options = {
         '%=',
         "%{% &showcmdloc == 'statusline' ? '%-10.S ' : '' %}",
         "%{% exists('b:keymap_name') ? '<'..b:keymap_name..'> ' : '' %}",
+        "%{% &busy ? '<busy> ' : '' %}",
```

Comment:
A single-char default would be ideal, to avoid using more than 1 column. "◐" seems good unless we find something better, it's (1) visually distinct, (2) looks like a half-full process.
```suggestion
        "%{% &busy ? '◐ ' : '' %}",
```